packages/core/src/diff/changes/change.ts

```diff
@@ -624,6 +676,8 @@ export type DirectiveUsageEnumAddedChange = {
   meta: {
     enumName: string;
     addedDirectiveName: string;
+    addedToNewType: boolean;
+    directiveRepeatedTimes: number;
```

Comment:
My thought from seeing this property here for the first time: It is not obvious to what it means. Maybe it could be named better, or a comment you give some context. I am sure when I get to looking T the actual implementation it will make sense though.